src/pushsource/_impl/helpers.py

```diff
@@ -1,6 +1,14 @@
+import sys
+
 import six
 
 
+if sys.version_info.major < 3:
```

Comment:
In the other file where urlparse is used, the python2 vs python3 topic is handled via importing from six.moves.  Can you please do that here as well (if it works)?

Reply:
Replaced with `from six.moves.urllib.parse import urlparse, ParseResult`.